src/aggregator/aggregator/generic_elem.go

```diff
@@ -737,13 +741,19 @@ func (e *GenericElem) processValueWithAggregationLock(
 					flushLocalFn(e.FullPrefix(e.opts), e.id, e.TypeStringFor(e.aggTypesOpts, aggType),
 						point.TimeNanos, point.Value, lockedAgg.aggregation.Annotation(), e.sp)
 				}
+
+				if !lockedAgg.flushed {
+					e.forwardLagMetric(resolution, "local", flushType).
```

Comment:
What is this "local"?

Reply:
This indicates a local flush vs the remote flush for the forwarded case